### ISSUE TITLE
Does this support RX560?

### ISSUE THREAD

**Issue by rumham8** [Optiminer/OptiminerZcash#69]
$ ./optiminer-zcash --list-devices
[2018-01-15 11:09:55.671] [info] Optiminer/Zcash 1.7.0 (C) Optiminer 2017
Platform (-c) 0, Device (-d) 0:
  Platform Vendor: AMD
  Device Name: gfx803
  Driver Version: 2545.0 (HSA1.1,LC)
  Address bits: 64
  Available memory: 4294967296
[2018-01-15 11:09:55.849] [warning] Unknown AMD device 'gfx803'
  Has device specific faster kernel: No

Thanks

**Comment by Optiminer** [comment]
No, that's not supported. Out of curiosity, what Linux and drivers are you using? I had problems getting these new cards running at all.

Note, future development of Optiminer happens at
https://github.com/Optiminer/OptiminerEquihash

**Comment by rumham215** [comment]
Im using amdgpu-pro 17.50
